lms/templates/admin/instance.html.jinja2

```diff
@@ -49,6 +49,7 @@
     <fieldset class="box">
         {{ macros.settings_textarea(instance, "Notes", "hypothesis", "notes") }}
         {{ macros.disabled_text_field("Consumer key", instance.consumer_key) }}
+        {{ macros.disabled_text_field("Shared secret", instance.shared_secret) }}
```

Comment:
Without this the journey doesn't make a huge amount as you can't complete the LTI 1.1 journey